pkgs/by-name/py/pywalfox-native/package.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, python3, fetchPypi }:
+
+python3.pkgs.buildPythonApplication rec {
+    pname = "pywalfox-native";
+    version = "2.7.4";
+
+    src = fetchPypi {
+      inherit version;
+      pname = "pywalfox";
+      hash = "sha256-Wec9fic4lXT7gBY04D2EcfCb/gYoZcrYA/aMRWaA7WY=";
+    };
+
+    pythonImportsCheck = [ "pywalfox" ];
+
+    meta = with lib; {
+      homepage = "https://github.com/Frewacom/pywalfox-native";
+      description = "Native app used alongside the Pywalfox addon";
+      mainProgram = "pywalfox";
+      license = licenses.mpl20;
+      maintainers = with maintainers; [ tsandrini ];
```

Comment:
please add `mainProgram = "pywalfox";`

Reply:
> please add `mainProgram = "pywalfox";`

It is there? Or? I was testing it via flakes (`nix run .#pywalfox-native`) or am I missing something else?

Reply:
```
[nix-shell:~/.cache/nixpkgs-review/pr-271023-3]$ ls ./results/pywalfox-native/bin/pywalfox 
./results/pywalfox-native/bin/pywalfox

```
https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md#deprecations-sec-release-2311-lib-deprecations

Reply:
@kirillrdy I'm sorry, I don't really understand what you are trying to say? `meta.mainProgram` has been set since the initial commit and I haven't changed that line since, running for example `nix run .#pywalfox-native light` works as expected. Also parsing `lib.getExe` on the derivation from REPL works.

Reply:
ah, sorry mistake is on my part, i didn't see mainProgram :-)

Reply:
Ahh, no problem, I'm grateful for the patience :smile: